.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
-        language_version: "3.10"
```

Comment:
You missed https://github.com/python-discord/snekbox/blob/f2f1193ffdf7efa282f09555a3dc1501bfdbc5dd/pyproject.toml#L67

Can you check if these settings are redundant? If so, let's remove it from pre-commit-config.

Reply:
Will test and report back

Reply:
As far as I could tell, it did manage to successfully read and use the target-version from here. That said, the target-version flag has very little effect on the output in general, so being certain is relatievly difficult.